core/storage/statistics/gae_models.py

```diff
@@ -881,6 +1082,24 @@ def _shard_answers(
         list, shard the answers such that a list of answer lists are returned.
         The first entry is guaranteed to contain all answers of the current
         answer list.
+
+        Args:
+            current_answer_list: list(dict). List of answer dicts each of which
+                is stored as Json blob.
+            current_answer_list_size: list(int). Number of bytes required
```

Comment:
Are you sure this is a list(int)?

Reply:
Sorry for this. It should be an int. Corrected this now.